### PR TITLE
add javascript.util@0.12.12 w/ npm auto-update

### LINKED ISSUE #9975: [Request] Add javascript.util.min.js
**Library name:** javascript.util.min.js
**Git repository url:*https://github.com/bjornharrtell/javascript.util/blob/master/dist/javascript.util.min.js*
**npm package url(optional):** 
**License(s):* The MIT License (MIT)*
**Official homepage:**
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by hare1039**
close #9975, cc @bjornharrtell

Pull request for issue: #9975
Related issue(s): # #

@pvnr0082t Please help me review this PR. Thanks.
Note that the file structure is diffenent before `v0.12.2`
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/bjornharrtell/javascript.util
 * Official website (optional, not the repository): https://bjornharrtell.github.io/javascript.util/doc/index.html
 * NPM package url (optional): https://www.npmjs.com/package/javascript.util
 * License and its reference: MIT license, ref: https://github.com/bjornharrtell/javascript.util/blob/master/license-notice.txt#L5
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 16
   - Count of watchers: 2
   - Count of forks: 8
 * NPM download stats (optional): 
   - Downloads in the last day: 322
   - Downloads in the last week: 2455
   - Downloads in the last month: 8824

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/javascript.util/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "java",
+    "javascript",
+    "util",
+    "collection"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/bjornharrtell/javascript.util.git"
+  },
+  "homepage": "https://bjornharrtell.github.io/javascript.util/doc/index.html",
```
Comment (pvnr0082t) [inline ajax/libs/javascript.util/package.json]: @hare1039 We can use the shorter url `https://bjornharrtell.github.io/javascript.util/doc`.

**Comment by pvnr0082t** [comment]
@hare1039 Correct your note, the file structure is different before v0.12.2 because there is no `dist` folder and pre-built files. BTW, npm auto-update can't get v0.12.6. :+1:

**Comment by hare1039** [comment]
@pvnr0082t Sorry for the misleading note. And I've updated the PR, please review this PR again. Thanks.

**Comment by kennynaoh** [comment]
LGTM

**Pull request merged**